### DISCUSSION TITLE
How can i set request resources in k8s for collector?

### DISCUSSION THREAD

**Discussion by zelanastasia** [jaegertracing/jaeger-operator#2043] [Q&A] [answered]
Hello, i get warning:
Event occurred" object="jaeger-operator/simple-prod-collector" kind="HorizontalPodAutoscaler" apiVersion="autoscaling/v2beta2" type="Warning" reason="FailedGetResourceMetric" message="failed to get cpu utilization: missing request for cpu"

I've noticed that in collector pod description not set resources? Then what limits in resources do we have?

**Comment by frzifus** [comment, marked as answer]
Hi @zelanastasia, resources should be configurable in `JaegerCollectorSpec.Resources`. Its part of the [embedded JaegerCommonSpec](https://github.com/jaegertracing/jaeger-operator/blob/58f91556e182a4fd929f1abbd0522b1e560e51b3/apis/v1/jaeger_types.go#L421).

https://github.com/jaegertracing/jaeger-operator/blob/58f91556e182a4fd929f1abbd0522b1e560e51b3/apis/v1/jaeger_types.go#L210

I thought we would assign default values to the collector. This is apparently not correct. :octocat: 
https://github.com/jaegertracing/jaeger-operator/blob/58f91556e182a4fd929f1abbd0522b1e560e51b3/pkg/deployment/collector.go#L213

**Reply by zelanastasia** [reply]
Yeah, i already find  anwser too, it was in doc: https://www.jaegertracing.io/docs/1.37/operator/#production-strategy
Thnx
